torch/testing/_internal/common_methods_invocations.py

```diff
@@ -538,6 +538,7 @@ def method_tests():
         ('add', (), ((S, S, S),), 'scalar_broadcast_lhs', (True,)),
         ('add', (S, S, S), (3.14,), 'constant', (True,)),
         ('add', (), (3.14,), 'scalar_constant', (True,)),
+        ('add', (S, S, S), (3.14j,), 'complex_scalar_constant', (True,)),
```

Comment:
Does this actually calls into the scalar overload? I think that the python binding is converting these to Tensors all the time.

You can double check this by doing: 
```python
import torch

a = torch.rand(10, requires_grad=True)
print((a + a).grad_fn)
print((a + 1.).grad_fn)
```
That will print 
```
<AddBackward0 object at 0x7fc1c1446150>
<AddBackward0 object at 0x7fc1c2f14f50>
```

The important part is the `0` here that means that both are actually using the 0th overload (the one for Tensor inputs).